tests/data/vmware-sdwan.json

```diff
@@ -0,0 +1,2571 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.8072.3.2.10",
+                    "sysDescr": "VeloCloud EDGE610",
+                    "sysContact": "<private>",
+                    "version": null,
+                    "hardware": "EDGE610",
+                    "features": null,
+                    "os": "vmware-sdwan",
+                    "type": "appliance",
+                    "serial": null,
```

Comment:
Same as above, but doubt it?

Reply:
Same here, no serial via snmp.. can provide the raw snmpwalk if needed, but sadly they don't provide that data.